src/apps/pages/sections/FreeText.astro

```diff
@@ -9,5 +10,21 @@ const { content } = Astro.props;
 >
   <TinaMarkdown
     content={content}
+    components={richTextComponents}
```

Comment:
Interesting pull this out into a utility class. Could you explain a little about why we didn't just inline the component?